add_glyphs.py

```diff
@@ -158,10 +161,17 @@ def add_glyph_data(font, seqs, seq_to_advance, vadvance):
   #
   # The added codepoints have no advance information, so will get a zero
   # advance.
+  #
+  # If a glyf table is present, empty glyphs will be added to ensure
+  # compatibility with systems requiring a glyf table, like Windows 10.
 
   cmap = get_font_cmap(font)
   hmtx = font['hmtx'].metrics
   vmtx = font['vmtx'].metrics
+  if 'glyf' in font:
+    pen = TTGlyphPen(None)
+    empty_glyph = pen.glyph()
+    glyf = font['glyf']
```

Comment:
I would add an `else: glyf = None` after this, then later when you add the empty glyphs you check `if glyf is not None:`.

Reply:
Good call, thanks!